spec/core/ics-003-connection-semantics/README.md

```diff
@@ -238,12 +238,76 @@ function verifyNextSequenceRecv(
     return client.verifyNextSequenceRecv(connection, height, connection.delayPeriodTime, connection.delayPeriodBlocks, connection.counterpartyPrefix, proof, portIdentifier, channelIdentifier, sequence, nextSequenceRecv)
 }
 
+function verifyMultihopMembership(
+  connection: ConnectionEnd,
+  height: Height,
+  proof: MultihopProof,
+  connectionHops: String[],
+  key: String,
+  value: ArrayBuffer) {
+    multihopConnectionEnd = abortTransactionUnless(getMultihopConnectionEnd(proof))
```

Comment:
Could we add some comments here and clarify in particular what `connection` is vs the `multihopConnectionEnd`.

Reply:
done